tests/ZendTest/Mvc/Controller/Plugin/FilePostRedirectGetTest.php

```diff
@@ -448,7 +448,7 @@ public function testCorrectInputDataMerging()
                     ),
                 ),
                 1 => array(
-                    'text' => null,
+                    'text' => '',  // #6497: Input is present but empty
```

Comment:
I'd prefer to see this as a separate case, to ensure the original `null` value case continues to pass.